app/segments/drill/[id]/submission/input.js

```diff
@@ -136,7 +153,114 @@ async function uploadAttempt(
   }
 }
 
-//TODO: Create a function to check leaderboard and update if needed
+//A function to check leaderboard and update if needed
+function handleLeaderboardUpdate(
+  uploadData,
+  drillInfo,
+  currentLeaderboard,
+  teamId,
+  drillId,
```

Comment:
use uploadData.did

Reply:
I passed in just uploadData (which also contains did) as suggested in another comment